src/mds/MDSAuthCaps.h

```diff
@@ -45,16 +46,19 @@ struct MDSCapSpec {
   static const unsigned SET_VXATTR	= (1 << 3);
   // if the capability permits mksnap/rmsnap
   static const unsigned SNAPSHOT	= (1 << 4);
+  // if the capability permits to bypass osd full check
+  static const unsigned FULL	        = (1 << 5);
```

Comment:
Should `MDSCapParser` be updated to parse the new cap?  Currently the only way to get it is through `*` or `all`.